scan/src/pages/ValidatorHomePage.re

```diff
@@ -596,7 +612,18 @@ let make = () => {
         </Col>
       </Row>
     </div>
-    <ValidatorList allSub />
+    <div className=Styles.searchContainer>
+      <input
+        type_="text"
+        className=Styles.searchBar
+        placeholder="Search Validator"
+        onChange={event => setSearchTerm(ReactEvent.Form.target(event)##value)}
```

Comment:
setState function expects the function that returns a new state.
So, It should be like this.
```
event => {
      let newVal = ReactEvent.Form.target(event)##value;
      setSearchTerm(_ => newVal);
}
```